draft-ietf-quic-http.md

```diff
@@ -794,6 +815,37 @@ the server MAY send another GOAWAY frame with an updated last stream identifier.
 This ensures that a connection can be cleanly shut down without losing requests.
 
 
+# MAX_PUSH_ID {#frame-max-push-id}
+
+The MAX_PUSH_ID frame (type=0xA) is used by clients to control the number of
+server pushes that the server can initiate.  This sets the maximum value for a
+Push ID that the server can use in a PUSH_PROMISE frame.  Consequently, this
+also limits the number of push streams that the server can initiate.
```

Comment:
I'm not sure I'm following this completely. Clearly MAX_PUSH_ID limits the number of *promises* that the server can make. But isn't the number of actual push streams limited by the MAX_STREAM_ID? 

(This makes me wonder if "Promise ID" might be a better name than "Push ID" but I may well just be confused.)